### ISSUE TITLE
Difference between initial bootstrapped files and current bootstrapped wheels

### ISSUE THREAD

**Issue by kushaldas** [freedomofpress/securedrop-debian-packaging#251]
If we check the [bootstrap](https://github.com/freedomofpress/securedrop-debian-packaging/tree/main/bootstrap) (from https://github.com/freedomofpress/securedrop-debian-packaging/commit/cc932321b65018433aa9a6ee2c15a3648df99c56)  directory and the [sha256sums](https://github.com/freedomofpress/securedrop-debian-packaging/blob/048e0a1d4f398bfcb24fde77ae86b15a97c6231c/bootstrap-sha256sums.txt) file, then we will notice there are two extra packages. I am wondering if they are at all required in the bootstrapping process.

```diff
+3b0769bd65ca00ffcb57975e750fac079639a3d3ce393c5a0d4c65123141c87a  click-7.1.2-py2.py3-none-any.whl
+d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a  click-7.1.2.tar.gz
+bab8000c26c590c10d33a40cf524cc65df925224c556b429b666b3478158f2b9  pip_tools-6.0.1-py3-none-any.whl
+3b0c7b95e8d3dfb011bb42cb38f356fcf5d0630480462b59c4d0a112b8d90281  pip-tools-6.0.1.tar.gz
```

@conorsch anything comes to your mind?

I found this while figuring out the steps for `Focal` based work.

**Comment by conorsch** [comment]
I'm not sure I understand the problem you're describing. Are you suggesting that the dependencies you list above should be removed from the bootstrapping logic? If I run `./scripts/build-sync-wheels --cache ./bootstrap -p $PWD` (from the README) locally, part of the output shows:

```
Collecting click==7.1.2
  Using cached click-7.1.2.tar.gz (297 kB)
[...snip...]
Collecting pip-tools==6.0.1
  Using cached pip-tools-6.0.1.tar.gz (120 kB)
```

So those packages are indeed being used. After that command completes, `git status` shows no changes on in my working copy. Are you seeing something different?

**Comment by kushaldas** [comment]
@conorsch I think I finally found the cause:

https://github.com/freedomofpress/securedrop-debian-packaging/commit/cc932321b65018433aa9a6ee2c15a3648df99c56 here you added `pip-tools` in the dependency, but I guess you forgot to update the `bootstrap-sha256sums.txt` file afterwards. 

I will update the file in the coming PR.

**Comment by legoktm** [comment]
This was eventually fixed in e7224f91646f849564bfcf5a6f0ac5ec57dadc2e by @cfm.
